DNN Platform/DotNetNuke.Abstractions/Portals/IPortalAliasInfo.cs

```diff
@@ -1,27 +1,59 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
 namespace DotNetNuke.Abstractions.Portals
 {
-    using System.Data;
-    using System.Xml;
-    using System.Xml.Schema;
+    using DotNetNuke.Abstractions.Urls;
 
+    /// <summary>
+    /// Portal Alias Info interface.
+    /// </summary>
     public interface IPortalAliasInfo
     {
-        //BrowserTypes BrowserType { get; set; }
-        string CultureCode { get; set; }
+        /// <summary>
+        /// Gets or sets the http alias.
+        /// </summary>
         string HTTPAlias { get; set; }
-        bool IsPrimary { get; set; }
-        int KeyID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal Alias ID.
+        /// </summary>
         int PortalAliasID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal ID.
+        /// </summary>
         int PortalID { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether if the portal is the primary portal.
+        /// </summary>
+        bool IsPrimary { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the poral is a redirect portal.
+        /// </summary>
         bool Redirect { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Browser Type.
+        /// </summary>
+        BrowserTypes BrowserType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the culture code.
+        /// </summary>
+        string CultureCode { get; set; }
+
+        /// <summary>
+        /// Gets or sets the skin.
+        /// </summary>
         string Skin { get; set; }
 
-        void Fill(IDataReader dr);
-        XmlSchema GetSchema();
-        void ReadXml(XmlReader reader);
-        void WriteXml(XmlWriter writer);
+        /// <summary>
+        /// Gets or sets the key ID.
+        /// </summary>
+        int KeyID { get; set; }
```

Comment:
This member doesn't seem to be necessary for this interface.  Am I missing something?

Reply:
`KeyID` is a property for the [`IHydratable`](https://github.com/dnnsoftware/Dnn.Platform/blob/cc61549005ab5ea1c66fa91670931642a62893e4/DNN%20Platform/Library/Entities/Modules/IHydratable.cs#L10) interface.

Reply:
I agree, I don't think it should be included either. I'll remove it in the next update